crates/flipperzero/src/furi/thread.rs

```diff
@@ -1,7 +1,194 @@
 //! Furi Thread API.
 
+#[cfg(feature = "alloc")]
```

Comment:
Since this can't really be used without `alloc`, we could move the `#[cfg(feature = "alloc")]` to the module definition in `furi/mod.rs`.

Reply:
A few standalone functions can be used without `alloc` (e.g. `thread::sleep`).

Reply:
Ah. Sorry. I was sleepy last night.